FlowCrypt/Core/CoreTypes.swift

```diff
@@ -139,13 +139,13 @@ struct SendableMsg: Equatable {
 }
 
 extension SendableMsg {
-    func copy(body: SendableMsgBody, atts: [Attachment], pubKeys: [String]?) -> SendableMsg {
+    func copy(body: SendableMsgBody, atts: [Attachment], pubKeys: [String]?, bcc: [String]) -> SendableMsg {
```

Comment:
I think it'll be better to use `includeBcc: Bool = true` param, instead of `bcc: [String]`.
So in common case we won't need to pass additional `bcc` array. And if `bcc` should be removed - you just set `includeBcc: false` parameter.

Reply:
Sounds good!